__tests__/shutdown.test.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/* eslint-env jest */
+
+const fs = require('fs')
+const {promisify} = require('util')
+const Bot = require('../index.js')
```

Comment:
I think we should import `Bot` from `lib` instead of the transpiled `index.js`. Jest code is already going through babel, so it doesn't make a lot of sense to do this twice and require us to run `yarn build` before we can `yarn test`.

Also since the jest code is compiled by babel, we can use es6 imports instead of node's `require` syntax!

Reply:
cool, I'll change this on all the bots

Reply:
slight change, rather than `import Bot from '../lib/bot.js'` I did `import Bot from '../lib/entry.js'` because requiring entry matches what the module will be.